src/aap_eda/settings/default.py

```diff
@@ -422,3 +422,5 @@ def _get_secret_key() -> str:
     "ansible_base.authentication.backend.AnsibleBaseAuth",
     "django.contrib.auth.backends.ModelBackend",
 ]
+
+EDA_PG_NOTIFY_DSN = "host=localhost dbname=eda user=postgres password=secret"
```

Comment:
The values should be taken from the DB config. It is a low effort and provides flexibility in future changes and tests.

Reply:
we can move this line to the right place later

Reply:
@Alex-Izquierdo Why would it come from DB CONFIG if they have 2 different databases with different config options why would you want to tie it our DB. One DB could be using SSL other could be using password.

Reply:
I'm not discussing about the place, the settings seems the right place. I'm talking about not having hardcoded the values, this would not even work in docker-compose where the hostname is different. It must be taken from the variables.

Reply:
@mkanoor precisely because we might have a second DB that would be configured just for this, it must be taken from that DB config. I assume that right now we are going to work with the same DB. We can add a second one later. But it shoud be taken from the variables, as it is now not even would work in minikube/docker-compose dev envs because the hostname is different.

Reply:
We can add a new database for this in `DATABASES` (with a fallback into our current default DB) or set it to allow to be configurable by the user:
```
_PG_NOTIFY_DSN = f"{our string composed from DATABASES}"
PG_NOTIFY_DSN = settings.get("PG_NOTIFY_DSN", _PG_NOTIFY_DSN)
```  

but at least for now, as minimal, we need to compose the variable from the current DB settings:
```
PG_NOTIFY_DSN = f"host={DATABASES["default"]["HOST"] etc etc..."
```

Note that we don't need the "EDA" prefix, that would be only for environment variables and is added automatically by dynaconf.

Reply:
solved